### PR TITLE
[srvls][SRVKE-801] Add KafkaChannel note to docs

### PR DESCRIPTION
**Pull request by abrennan89**
Applies for 4.6+

### CODE REVIEW

**Review by cardil — APPROVED** [review]
/lgtm

from QE

File: modules/serverless-rn-1-14-0.adoc [hunk 1]
```diff
@@ -18,7 +18,7 @@
 +
 [IMPORTANT]
 ====
-Only the `v1beta1` version of the API for Knative Kafka on {ServerlessProductName} is supported. Do not use the `v1alpha1` version of the API, as this is deprecated.
+Only the `v1beta1` version of the APIs for `KafkaChannel` and `KafkaSource` objects on {ServerlessProductName} are supported. Do not use the `v1alpha1` version of these APIs, as this version is now deprecated.
```
Comment (matzew) [inline modules/serverless-rn-1-14-0.adoc:21]: Perfect!

**Review by matzew — APPROVED** [review]
/lgtm
/approve

Engineering

**Comment by bmcelvee** [comment]
LGTM

**Pull request merged**